### PR TITLE
[zh] Sync platform-setup/openshift

### PR DESCRIPTION
**Pull request by ii2day**
Please provide a description for what this PR is for.

>content/zh/docs/setup/platform-setup/openshift/index.md

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by windsonsea** [comment]
Please sign CLA to move the flow forward.

**Comment by ii2day** [comment]
> Please sign CLA to move the flow forward.

done

**Review by my-git9 — COMMENTED** [review]
/ok-to-test

File: content/zh/docs/setup/platform-setup/openshift/index.md [hunk 1]
```diff
@@ -11,16 +11,6 @@ owner: istio/wg-environments-maintainers
 test: no
 ---
 
-根据以下操作指南 为 Istio 准备一个 OpenShift 集群。
```
Comment (windsonsea) [inline content/zh/docs/setup/platform-setup/openshift/index.md:14]: line 14 shall be remained.

**Review by windsonsea — COMMENTED** [review]
It would be better for reviewers if you can put the corresponding en PR #13795 in the desc.

**Comment by windsonsea** [comment]
Note the lint error:

```
./zh/docs/setup/platform-setup/openshift/index.md:23: MD047 File should end with a single newline character
```

**Comment by ii2day** [comment]
> Note the lint error:
> 
> ```
> ./zh/docs/setup/platform-setup/openshift/index.md:23: MD047 File should end with a single newline character
> ```

done

**Review by windsonsea — APPROVED** [review]
Thanks, LGTM.

**Pull request merged**